src/components/network/NetworkItem.vue

```diff
@@ -27,15 +62,20 @@
       >
         <i class="material-icons notranslate">check</i>
       </div>
+      <div v-else style="display: block; width: 24px;"></div>
```

Comment:
what is going on here?

Reply:
😅  the problem was to vertically align the powered by things. When the check is not present, there is an empty space I could not fill elseways

Reply:
this should be deleted.